### ISSUE TITLE
all live code links hosted on rnplay.org are dead 

### ISSUE THREAD

**Issue by ixje** [browniefed/react-native-animation-book#14]
see the title.

This concerns at least the `setOffset` and `flattenOffset` sections under `Animated.ValueXY`

**Comment by asiaantczak** [comment]
all links above are working, please close the issue

**Comment by ixje** [comment]
@asiaantczak are you saying that the link here works for you?
http://browniefed.com/react-native-animation-book/api/SETOFFSET.html

Because if I try to go to: [https://rnplay.org/apps/m_t0yQ](https://rnplay.org/apps/m_t0yQ) on that page then I get a clear "Page not Found".

**Comment by asiaantczak** [comment]
@ixje sorry, you are right; however you haven't been very specific in what was not working - as links in setOffset and flattenOffset sections work fine, just Live Code sections on both are not working;

**Comment by ixje** [comment]
@asiaantczak no worries. I do think that the subject can't be more specific than what it already is. It describes exactly which links don't work (as in; any hyperlink pointing to rnplay.org, which are used for the "live code" samples).

**Comment by asiaantczak** [comment]
you are right, not looked properly on the title, but focused on the description; sorry!
